Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -719,18 +723,23 @@ def list_indicators(self, indicator_id: Optional[str] = None) -> List:
         if resp.status_code == 404:
             return []
         resp = resp.json()
+        results.update(resp)
+
+        while next_link := resp.get('@odata.nextLink'):
+            resp = self.indicators_http_request('GET', full_url=next_link, url_suffix=None, timeout=1000)
```

Comment:
lets break the loop once we reached the limit so we wont do unnecessary iterations